presto-main/src/main/java/com/facebook/presto/sql/relational/FunctionResolution.java

```diff
@@ -371,4 +371,10 @@ public boolean isWindowValueFunction(FunctionHandle functionHandle)
     {
         return windowValueFunctions.contains(functionAndTypeResolver.getFunctionMetadata(functionHandle).getName());
     }
+
+    @Override
+    public FunctionHandle lookUpFunction(String functionName, List<Type> inputTypes)
```

Comment:
Let's call this `lookupBuiltInFunction` to make clear this does not support session functions or functions supplied by a function namespace manager.